common/src/main/java/com/microsoft/identity/common/internal/fido/AuthFidoChallenge.kt

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.internal.fido
+
+/**
+ * An object representing an auth FIDO challenge request.
+ */
+data class AuthFidoChallenge(
+    override val challenge: String,
+    override val relyingPartyIdentifier: String,
+    override val userVerificationPolicy: String,
+    override val version: String,
+    override val submitUrl: String,
+    override val keyTypes: List<String>,
+    override val context: String,
+    /**
+     * List of credential IDs the client will use for filtering.
+     */
+    val allowedCredentials: List<String>
+) : IFidoChallenge
```

Comment:
nit: new lines at the end of the new files

Reply:
done

Reply:
Interesting, I did put an extra newline at the end of all the classes, but that doesn't seem to be reflected in the latest commit. Please take my word for it that I did it :)